templates/amp/entry.php

```diff
@@ -7,6 +7,9 @@
 	$share_link     = $this->get( 'share_link' );
 	$update_time    = $this->get( 'update_time' );
 	$share_link_amp = $this->get( 'share_link_amp' );
+
+	/* This filter is defined in class-wpcom-liveblog-amp.php */
+	$facebook_app_id = apply_filters( 'liveblog_amp_facebook_share_app_id', false );
```

Comment:
Considering how this is going to be used, it feels like a default value of `''` empty string, would be in keeping the right type at least.